apps/functions-subscription/api/internal.yaml

```diff
@@ -192,12 +192,12 @@ paths:
             schema:
               $ref: '#/components/schemas/CreateTrial'
       responses:
-        '201':
-          description: Created
+        '202':
+          description: Accepted
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Trial'
+                $ref: '#/components/schemas/CreatedTrial'
```

Comment:
**thought**: How about returning `Trial` instead of creating a new type `CreatedTrial`? The only difference is the name, and all the other fields are optional